build/build-image/cross/Dockerfile

```diff
@@ -79,15 +79,12 @@ RUN export ETCD_VERSION=v3.0.14; \
   && ln -s ../src/etcd/etcd-${ETCD_VERSION}-linux-amd64/etcd /usr/local/bin/
 
 # TODO: Remove the patched GOROOT when we have an official golang that has a working arm and ppc64le linker
-ENV K8S_PATCHED_GOLANG_VERSION=1.7.4 \
+ENV K8S_PATCHED_GOLANG_VERSION=go1.8beta2 \
```

Comment:
Yes, I really hope it will. That's my intention and goal.
We have to validate it and do the hard work with the betas though